### DISCUSSION TITLE
[Vector Problem]向量首尾不在正确位置

### DISCUSSION THREAD

**Discussion by suucss** [3b1b/manim#1532] [Q&A] [answered]
如图，向量首尾并不完全在(0,0)和(3,2)，我应该如何做？谢谢您！
The following picture showed that the vector doesn`t properly start at (0,0) and end at (3,2), what should I do to solve it? Thank you very much !

> from manim import *
class Animation(Scene):
    def construct(self):
        plane=NumberPlane()
        self.play(Create(plane,run_time=2)),self.play(Write(plane.get_axis_labels()))
        arr1 = Arrow([0,0,0],[3,2,0],color=RED)
        self.play(GrowArrow(arr1))
        self.wait(2)

![problem](https://user-images.githubusercontent.com/84609589/121008402-8a6fce80-c7c5-11eb-81f5-e656e6732008.png)

**Comment by nakaken88** [comment, marked as answer]
I think it will work.

`arr1 = Arrow([0,0,0],[3,2,0],color=RED,buff=0)`

The default buff value of Arrow is MED_SMALL_BUFF, so there was the gap.

**Reply by suucss** [reply]
Oh, your advice WORKS!!! Thank you very much!
